pallets/orml-currencies-allowance-extension/src/benchmarking.rs

```diff
@@ -0,0 +1,29 @@
+#![allow(warnings)]
+use super::{Pallet as TokenAllowance, *};
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+use sp_std::prelude::*;
+
+benchmarks! {
+	add_allowed_currencies {
+		let nativeCurrencyId = <T as  orml_currencies::Config>::GetNativeCurrencyId::get();
```

Comment:
this and the following should be snake case, ie `native_currency_id`

Reply:
Done